.eslintrc.js

```diff
@@ -1,8 +1,16 @@
 module.exports = {
-  extends: '@cmsgov/eslint-config-design-system',
+  extends: ['@cmsgov/eslint-config-design-system'],
   rules: {
     // Avoid exploits. If you need dangerouslySetInnerHTML, then temporarily
     // disable this rule in the script rather than removing it from here.
     'react/no-danger': 'error',
   },
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      parser: '@typescript-eslint/parser',
+      extends: ['plugin:@typescript-eslint/recommended'],
+      rules: {},
+    },
+  ],
```

Comment:
Not sure if it's related to this, but `yarn lint` command fails with this error:
```
[08:42:06] 🔎  Linting files in: ./ and ignoring paths: **/node_modules/**, **/dist/**, **/helpers/**, **/__tests__/**, tmp/**, **/types/**, examples/create-react-app-typescript/**

events.js:292
      throw er; // Unhandled 'error' event
```

Reply:
i’ll take a look at this soon